calblueprint/meepanyar
# Add tariffPlans patch

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
Sorry for the large diff. We shouldn't try to change the style of `schema.js` since that breaks the patches (they rely on diffs). The patches rely on the generated files' default styles.

You can ignore the large diff in `schema.js`. It's caused by regenerating the schema.

Added patch for schema generator that adds the `tariffPlans` field.
[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
1. Run `npm generate-schema`
2. Run `git status` and see that there are diffs
3. Check that `tariffPlans` is no longer a field in the `Sites` object of `schema.js`
4. Run `npm apply-schema-patches`
5. Check that `tariffPlans` is now a field in the `Sites` object of `schema.js`
6. Check that `git status` returns no diffs

[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
